docs/LLS_Metallicities-Wotta/MCMC_initial_guesses-full_list.dat

```diff
@@ -0,0 +1,5 @@
+##Run_now?  Sightline                 met_guess    dens_guess   carbalpha_guess   carbalpha   logUconstraint   UVB    Notes
```

Comment:
Yes, they are example files.  Initial guesses for the MCMC ball and a couple of other options.